docs/getting-started/overview.md

```diff
@@ -116,6 +116,16 @@ For information about extending the functionality of Zowe CLI by installing plug
 
 </details>
 
+### Zowe Client Software Development Kits (SDKs)
+
+The Zowe Client SDKs consist of programamatic APIs that you can use to build client applications or scripts that interact with z/OS. The following SDKs are available:
+ - Zowe Node Client SDK
+ - Zowe Python Client SDK
```

Comment:
Should we also mention the [Zowe Swift Client SDK](https://github.com/zowe/zowe-client-swift-sdk)?

Reply:
Not yet @tjohnsonBCM, Bauer recommnends that we hold off and add it later.